main/main.cpp

```diff
@@ -768,6 +769,9 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	packed_data->add_pack_source(zip_packed_data);
 #endif
 
+	// Initialize rng before argument parsing so that --seed can override it
+	Math::randomize();
```

Comment:
`randomize()` is already called implicitly on startup on 4.0, so I'd remove that other initialization to prevent it from being initialized twice.

Reply:
Are you referring to the removed line in `SceneTree::SceneTree()`? I tried commenting this line out, but it resulted in a fixed seed, so there doesn't appear to be anywhere else where the rng is initialized.

Reply:
I didn't notice you removed the other `randomize()` call, so this should be good to go.